src/components/Card/CourseCard.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react'
+import PlayButton from '../Button/PlayButton'
+import Card from './'
+
+const expansions = [false, 'horizontal', 'vertical']
+
+export const CourseMeta = ({meta}) => {
```

Comment:
If a component is being exported + added to living style guide examples, it should be its own file/folder IMO